torch/csrc/api/include/torch/nn/functional/loss.h

```diff
@@ -25,6 +25,22 @@ inline Tensor cosine_embedding_loss(
   return torch::cosine_embedding_loss(
       input1, input2, target, options.margin(), options.reduction());
 }
+  
+inline Tensor triplet_margin_loss(
+    const Tensor& anchor,
+    const Tensor& positive,
+    const Tensor& negative,
+    const TripletMarginLossOptions& options) {
```

Comment:
Since all `TripletMarginLossOptions`'s arguments are optional, we can provide an empty default value for `options` here:
```suggestion
    const TripletMarginLossOptions& options = {}) {
```
to support the `triplet_margin_loss(anchor, positive, negative)` use case. (We should add a test case for that as well.)